NitroxClient/GameLogic/EscapePodManager.cs

```diff
@@ -30,8 +30,13 @@ public EscapePodManager(IPacketSender packetSender, IMultiplayerSession multipla
 
         public void AssignPlayerToEscapePod(EscapePodModel escapePod)
         {
+            if (escapePod != null)
```

Comment:
Do we consider this a release blocker?  How often is it happening?

Reply:
It doesn't happen often so... Not really a Release blocker